app/policies/photo_policy.rb

```diff
@@ -0,0 +1,33 @@
+class PhotoPolicy < ApplicationPolicy
+  attr_reader :user, :photo
+
+  def initialize(user,photo)
+    @user = user
+    @photo = photo
+  end 
+
+  def show?
+    owner? ||
+    !photo.owner.private? ||
+    photo.owner.followers.include?(user)
+  end 
+
+  def create?
+    owner?
```

Comment:
I think that would break it. We are defining a policy to create a photo with a rule which is based on whether we are the owner of that photo ; but that photo does not exist yet.

Reply:
fixed! thank you!

updated to:
  def create?
    true
  end